openassessment/assessment/migrations/0025_auto__add_field_peerworkflow_cancelled_at.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+import datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
```

Comment:
One thought on this migration. There are many rows in the PeerWorkflow table in production. We should wrap this migration operation in a database lock to avoid having the migration fail. 

example https://github.com/edx/edx-ora2/blob/muzaffar/cancel-student-submission-tnl900/openassessment/assessment/migrations/0018_auto__add_field_assessmentpart_criterion.py#L17-17